app/scripts/app.js

```diff
@@ -32,12 +32,16 @@
       }
     });
     Idle.watch();
+    $rootScope.$on('IdleTimeout', function() {
+      $window.localStorage.clear();
+      $window.location='/';
+   });
   })
 
   .config(function($stateProvider, $urlRouterProvider, $httpProvider, $ionicConfigProvider, IdleProvider, KeepaliveProvider) {
     // configure Idle settings
     IdleProvider.idle(12); // in seconds
-    IdleProvider.timeout(3600); // in seconds
+    IdleProvider.timeout(600); // in seconds
```

Comment:
falso... el timeout es el tiempo en el que el usuario debe responder para q se considere timeout. el idle es el tiempo que pasa entre el timeout y el tiempo actual (que sería el cierre de sesión).  https://github.com/HackedByChinese/ng-idle/wiki/Idle-and-Idleprovider#idleprovider